Add python section to sign-into-web-app guide. OKTA-354778 (#1897)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/configure-packages/python/configmid.md

> Flask samples read configuration from file `client_secrets.json`.

Reviewer: ```suggestion
Flask samples read the configuration from `client_secrets.json`.
```

> Add file `client_secrets.json` to the folder with your application which include the following parameters:

Reviewer: ```suggestion
Add the file `client_secrets.json` to the folder with your application that includes the following parameters:
```

```bash
{
  "auth_uri": "https://{{OKTA_DOMAIN}}/oauth2/default/v1/authorize",
  "client_id": "{{CLIENT_ID}}",
  "client_secret": "{{CLIENT_SECRET}}",
  "redirect_uri": "http://localhost:8080/authorization-code/callback",
  "issuer": "https://{{OKTA_DOMAIN}}/oauth2/default",
  "token_uri": "https://{{OKTA_DOMAIN}}/oauth2/default/v1/token",
  "userinfo_uri": "https://{{OKTA_DOMAIN}}/oauth2/default/v1/userinfo"
}
```

> Then you can load config in your application:

Reviewer: ```suggestion
Then you can load the config in your application:
```
